dmeta/example/pom.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We shouldn't introduce so many `pom.xml` right?
One pom file `dmeta/pom.xml` is enough.